index.html

```diff
@@ -609,12 +609,21 @@ <h2>
           member is set.
         </p>
       </div>
-      <p data-link-for="PaymentRequest">
-        Because the simultaneous display of multiple <a>PaymentRequest</a> user
-        interfaces might confuse the user, this specification limits the
-        <a>user agent</a> to displaying one at a time via the <a>show()</a>
-        method. This is ensured by a <dfn>payment request is showing</dfn>
-        boolean.
+      <p>
+        A <var>request</var>'s <dfn>payment-relevant browsing context</dfn> is
+        that <a>PaymentRequest</a>'s <a data-cite=
+        "HTML#concept-relevant-global">relevant global object</a>'s browsing
+        context's <a>top-level browsing context</a>. Every <a>payment-relevant
+        browsing context</a> has a <dfn>payment request is showing</dfn>
```

Comment:
Hi @marcoscaceres,

The above sentence is quite dense. Is it correct? Should it be:

<pre>
       A <var>request</var>'s <dfn>payment-relevant browsing context</dfn> is
       that <a>PaymentRequest</a>'s <a data-cite=
        "HTML#concept-relevant-global">relevant global object</a>'s <a>top-level browsing context</a>.
</pre>

Or even better:

<pre>
    The <dfn>payment-relevant browsing context</dfn> of a <a>PaymentRequest</a>
    is the <a>top-level browsing context</a> of the <a data-cite=
        "HTML#concept-relevant-global">relevant global object</a>.
</pre>

Reply:
Prefer we leave this for now, tbh. It's convoluted, but accurate.